src/lib/components/PublishButton/SubmitReviewModal.js

```diff
@@ -29,13 +32,34 @@ export class SubmitReviewModal extends Component {
       onClose,
       onSubmit,
       publishModalExtraContent,
+      directPublish,
     } = this.props;
     const communityTitle = community.metadata.title;
+
+    let headerTitle, msgWarningTitle, msgWarningText1, submitBtnLbl;
```

Comment:
nit: could this part (here to line 54) be somehow better formatted or refactored? it is quite unreadable

Reply:
There is not much more that can be done. We have two options:
- global if and then assign values inside (as it is now)
- a ternary per variable, since the values are long there is not much readability gained and we add 4 ifs...